Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.10
```

Comment:
@addgene/platform-engineering - this is my naive attempt at modernizing openbio to use docker. Your critique will be much appreciated.

Openbio just needs a simple python environment, no frills (previously we had a  collection of shell commands to set it up)